meta-cli/src/cli/list.rs

```diff
@@ -0,0 +1,175 @@
+        let dir = args.dir()?;
+        let config_path = args.config.clone();
+
+        let config = Arc::new(Config::load_or_find(config_path, &dir)?);
+
+        let console = ConsoleActor::new(Arc::clone(&config)).start();
+
+        let action_generator = SerializeActionGenerator::new(
+            config.dir().unwrap_or_log().into(),
+            dir.clone().into(),
+            config
+                .prisma_migrations_base_dir(PathOption::Absolute)
+                .into(),
+            true,
+        );
+
+        let task_source = TaskSource::Discovery(dir.clone().into());
+
+        let mut init = TaskManagerInit::<SerializeAction>::new(
+            Arc::clone(&config),
+            action_generator,
+            console,
+            task_source,
+        );
+
+        if let Some(max_parallel_tasks) = self.max_parallel_loads {
+            init = init.max_parallel_tasks(max_parallel_tasks);
+        }
+
+        let report = init.run().await;
+
+        match report.stop_reason {
+            StopReason::Error => bail!("Task manager stopped due to error."),
+            StopReason::Manual | StopReason::ManualForced => {
+                bail!("Task manager stopped manually.")
+            }
+            StopReason::Natural => {}
+            StopReason::Restart => panic!("Restart is not supported for serialization."),
+        }
+
+        let typegraphs = report.into_typegraphs()?;
+
+        self.display_typegraphs(dir, typegraphs).await
+    }
+}
+
+#[derive(Debug, Clone, Deserialize)]
+struct TypegraphObject {
+    name: String,
+    url: String,
+}
+
+impl List {
+    async fn get_more_info(&self, dir: PathBuf) -> Result<Vec<TypegraphObject>> {
+        let query = r#"
+            query {
+                typegraphs {
+                    name
+                    url
+                }
+            }
+        "#;
+        let mut node_config = NodeConfig::default();
+        node_config.url = Url::parse("http://localhost:7891")?;
+        let node = node_config.build(dir).await?;
+        let response = node
+            .post("/typegate")
+            .unwrap()
+            .gql(query.into(), None)
+            .await?;
+
+        response
+            .data("typegraphs")
+            .map_err(|err| anyhow::anyhow!(err))
+    }
+
+    #[allow(clippy::vec_box)]
+    async fn display_typegraphs(
+        &self,
+        dir: PathBuf,
+        typegraphs: Vec<Box<Typegraph>>,
+    ) -> Result<()> {
+        let mut tables: Vec<Table> = Vec::new();
+
+        Table::print_table_header();
+
+        for tg in typegraphs {
+            let mut table = Table::new(tg.name().unwrap());
+            table.set_path(tg.get_path().ok());
+            tables.push(table);
+        }
+
+        if let Ok(more_info) = self.get_more_info(dir).await {
+            for tg in more_info {
+                let mut table = Table::new(tg.name);
+                table.set_url(Some(tg.url));
+                tables.push(table);
```

Comment:
/Merge entries with the same typegraph name?

Reply:
No, I just create table to display them